### DISCUSSION TITLE
Read color code

### DISCUSSION THREAD

**Discussion by p-levy** [kcleal/gw#68] [Q&A] [answered]
Hi Kez, 
I'm seeing some funky color reads in a sample when using gw that I don't when using IGV. Do you have any idea what's the reason? Is there a way to know the color/shade code?
Thanks!

![Screenshot 2024-12-20 at 12 28 07](https://github.com/user-attachments/assets/3cfb3a43-e22b-4ef4-8a1f-66dcd0805f2a)
![Screenshot 2024-12-20 at 12 28 38](https://github.com/user-attachments/assets/f25d6c34-7af1-4081-b360-86b574e0843b)

**Comment by kcleal** [comment]
Hi @p-levy,
I think the grey shaded bases could be N bases in the reads. It looks like GW has soft-clips turned on, but IGV does not?
This could be a bug in GW also. If you could send me this region as a bam file, I could take a look.

**Reply by p-levy** [reply]
Thanks for the fast answer! Soft clip is actually switched off in GW. Here's a bam extract. 
[output_region.bam.zip](https://github.com/user-attachments/files/18211007/output_region.bam.zip)

**Reply by kcleal** [reply]
Thanks. Will take a look soon - I'm away from a computer for a few days sorry

**Reply by p-levy** [reply]
Great for you! Have a nice Xmas break!

**Reply by p-levy** [reply]
Hi Kez, did you manage to have a look at this? Happy new year!

**Reply by kcleal** [reply]
Hi @p-levy,
Sorry I have not had a chance. I will take a look tomorrow, sorry about the delay.

**Reply by kcleal** [reply, marked as answer]
Hi @p-levy,

That should be fixed now - there was incorrect handling of the CREF_SKIP cigar operation. You will need to build gw from source whilst the next release is prepared - the next release will take a week or two.

**Reply by p-levy** [reply]
Thanks! I'll test it asap.
